kong/cmd/prepare.lua

```diff
@@ -0,0 +1,30 @@
+local prefix_handler = require "kong.cmd.utils.prefix_handler"
+local conf_loader = require "kong.conf_loader"
+local log = require "kong.cmd.utils.log"
+
+local function execute(args)
+  local conf = assert(conf_loader(args.conf, {
+    prefix = args.prefix
+  }))
+
+  local ok, err = prefix_handler.prepare_prefix(conf, args.nginx_conf)
+  if not ok then
+    log.verbose("could not prepare Kong")
+    error(err) -- report to main error handler
+  end
+end
+
+local lapp = [[
+Usage: kong prepare [OPTIONS]
+
+Prepares the Kong prefix in the configured prefix directory.
```

Comment:
The help message should also include an Example usage like the `kong compile` command does, to help the user understand the use-case that this command solves:
```
Prepare the Kong prefix in the configured prefix directory. This command can
be used to start Kong from the nginx binary without using the 'kong start' command.

Example usage:
  kong prepare -p /usr/local/kong -c kong.conf && nginx -p /usr/local/kong -c nginx.conf
```